core/templates/dev/head/css/oppia.css

```diff
@@ -5042,3 +5042,42 @@ md-card.preview-conversation-skin-supplemental-card {
   padding: 0;
   width: 1px;
 }
+
+.oppia-tooltip {
+  display: inline-block;
+  position: relative;
+}
+
+.oppia-tooltip-text {
+  background-color: #009688;
+  border-radius: 6px;
+  bottom: 97%;
+  color: #fff;
+  font-size: 0.6em;
+  padding: 5px;
+  position: absolute;
+  text-align: center;
+  visibility: hidden;
+  width: auto;
+  z-index: 1;
+}
+
+@media screen and (max-width: 800px) {
```

Comment:
Why is this necessary?

Reply:
I've added a down pointing arrow for the smaller screens through this. I didn't include it for larger screens as it was frequently getting misaligned.

Reply:
Yes let's remove it, to keep things as consistent as possible between devices.